### PR TITLE
Create digest helper function for meta tx forwarder

### PR DESCRIPTION
**Pull request by FHieser**
This solves this issue
https://linear.app/inverter/issue/SC-198/create-digest-helper-function-for-metatx-forwarder

### CODE REVIEW

**Review by marvinkruse — APPROVED** [review]
Looks good to me!

File: src/external/forwarder/TransactionForwarder.sol [hunk 1]
```diff
@@ -61,6 +76,28 @@ contract TransactionForwarder is
         }
     }
 
+    //--------------------------------------------------------------------------
+    // Internal
+
+    function getStructHash(ERC2771Forwarder.ForwardRequestData memory req)
```
Comment (Zitzak) [inline src/external/forwarder/TransactionForwarder.sol]: Should have an underscore to keep internal function convention

**Review by Zitzak — CHANGES_REQUESTED** [review]
Found only one small thing. Otherwise looks good!

File: src/external/forwarder/TransactionForwarder.sol (hunk 1 above)
Comment (FHieser) [reply src/external/forwarder/TransactionForwarder.sol]: @Zitzak added it accordingly
Will merge with the approve of Marvin K

**Pull request merged**